packages/origin-organization-irec-api/src/registration/registration.controller.ts

```diff
@@ -29,6 +31,17 @@ export class RegistrationController {
         );
     }
 
+    @Get('/:id')
+    @UseGuards(AuthGuard())
+    public getRegistrationsById(
+        @Param('id', new ParseIntPipe()) orgId: number,
+        @UserDecorator() loggedInUser: LoggedInUser
+    ): Promise<Registration[]> {
+        const isAdmin = loggedInUser.hasRole(Role.Admin, Role.SupportAgent);
+
+        return this.registrationService.find(isAdmin ? orgId.toString() : null);
+    }
+
```

Comment:
I'm not sure what is the intention of this method, but from what I see it alls admins to get registrations by organization Id and others to get all registrations (which is an obvious security risk).

This is actually the same as `public getRegistrations(@UserDecorator() loggedInUser: LoggedInUser)` above. 
The GET above will return all registrations for all organizations - which is fine - filter by org locally, there isn't many of those registrations expected to be, it's actually 1 by organization.